.github/CODEOWNERS

```diff
@@ -159,6 +159,8 @@
 
 /modules/services/unison.nix                          @pacien
 
+/modlues/services/window-managers/i3-sway/sway.nix    @alexarice
```

Comment:
Thanks for wanting to care for this module! It would be good to also add yourself in a `meta.maintainers` in the module itself, like so: https://github.com/rycee/home-manager/blob/0ee5c9536ba599af6728c7e9ec3219df3fb66feb/modules/home-environment.nix#L111